tests/gis_tests/geoapp/tests.py

```diff
@@ -450,7 +450,7 @@ def test_relate_lookup(self):
                 qs.count()
 
         # Relate works differently for the different backends.
-        if postgis or spatialite:
+        if postgis or spatialite or (mysql and connection.mysql_is_mariadb):
```

Comment:
This isn't very pretty. 

Is it worth adding a `mariadb` shortcut boolean to `../utils.py`?